src/angularODataUtils.ts

```diff
@@ -3,8 +3,8 @@ export class ODataUtils {
         const properties: string[] = [];
 
         for (const prop in obj) {
-            if (obj.hasOwnProperty(prop)) {
-                const value: string = this.quoteValue(obj[prop]);
+            if (obj.hasOwnProperty(prop) && obj[prop] !== undefined) {
```

Comment:
It this check enough? Or do you also need to check on `null` ?

Reply:
I think it's enough. Null need send to server.